cfme/common/provider.py

```diff
@@ -435,7 +435,6 @@ def _fill_smartstate_endpoint_dicts(self, provider_attributes):
 
     def _fill_vmrc_console_endpoint_dicts(self, provider_attributes):
         """Fills dicts with VMRC console endpoint data
-
```

Comment:
Can you remove this change? Kinda seems irrelevant to the PR.

Reply:
I guess it got introduced during rebase.